src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
Seems like we should name this something different since it relates to notifications and not just service worker. Also should we be add a feature detect for push manager?